extmod/asyncio/stream.py

```diff
@@ -17,7 +17,8 @@ async def __aenter__(self):
         return self
 
     async def __aexit__(self, exc_type, exc, tb):
-        await self.close()
+        self.s.close()
+        pass
```

Comment:
Why was await removed here? Also pass shouldn't be needed

Reply:
The `await` never made any sense here since the `close` method isn't async and in fact never was. It just works by coincidence.

You're right about the `pass` of course, I'll fix that up.